apps/csgo/scripts/server.conf

```diff
@@ -127,11 +127,12 @@ GAMEMODE=${GAMEMODE-"1"}
 ########################################
 
 # Slots for players
-SLOTS=${SLOTS-"10"}
+SLOTS=${SLOTS-"12"}
 
 # Additional slots for server admins and spectators
 # Note that reserved slots require sourcemod plugins.
-ADMIN_SLOTS=${ADMIN_SLOTS-"2"}
+# This will hide 2slots on the server browser(As example: 12 to 10) - it will still show 12slots(ingame on server via console status)
```

Comment:
That's not how it works right now, with SLOTS=12 and ADMIN_SLOTS=2, it will set sv_visiblemaxplayers "12" (see line 300) and -maxplayers 14 (see line 393). You can change the logic for that though, if you think that makes more sense.

Reply:
Ooh. Ops. I will look on it. If i can changes. Since it's on PR.